tests/test_basic_operation.py

```diff
@@ -102,25 +103,30 @@ def test_014_add_new_secondary():
     assert node3.wait_until_state(target_state="secondary")
     assert node2.wait_until_state(target_state="primary")
 
-
 def test_015_multiple_manual_failover_verify_replication_slot_removed():
-   monitor.failover()
-   assert node3.wait_until_state(target_state="primary")
-   assert node2.wait_until_state(target_state="secondary")
-   node2_replication_slots = node2.run_sql_query("select count(*) from pg_replication_slots")
-   assert node2_replication_slots == [(0,)]
-   node3_replication_slots = node3.run_sql_query("select count(*) from pg_replication_slots")
-   assert node3_replication_slots == [(1,)]
+    monitor.failover()
+    assert node3.wait_until_state(target_state="primary")
+    assert node2.wait_until_state(target_state="secondary")
+    node2_replication_slots = node2.run_sql_query("select count(*) from pg_replication_slots")
+    assert node2_replication_slots == [(0,)]
+    node3_replication_slots = node3.run_sql_query("select count(*) from pg_replication_slots")
+    assert node3_replication_slots == [(1,)]
    
-   monitor.failover()
-   assert node2.wait_until_state(target_state="primary")
-   assert node3.wait_until_state(target_state="secondary")
-   node2_replication_slots = node2.run_sql_query("select count(*) from pg_replication_slots");
-   assert node2_replication_slots == [(1,)]
-   node3_replication_slots = node3.run_sql_query("select count(*) from pg_replication_slots");
-   assert node3_replication_slots == [(0,)]
-                
+    monitor.failover()
+    assert node2.wait_until_state(target_state="primary")
+    assert node3.wait_until_state(target_state="secondary")
+    node2_replication_slots = node2.run_sql_query("select count(*) from pg_replication_slots");
+    assert node2_replication_slots == [(1,)]
+    node3_replication_slots = node3.run_sql_query("select count(*) from pg_replication_slots");
+    assert node3_replication_slots == [(0,)]
+
```

Comment:
That's a lot of review noise. I agree that we need to take care of this indentation issue at some point, but I'm leaning towards making it its own PR if that's possible. What do you think?